src/Common/RestApiClient.php

```diff
@@ -231,14 +236,20 @@ public function send($method, $path, $options = [], $withAuthorization = true)
      */
     public function raw($method, $path, $options, $withAuthorization = true)
     {
-        // By default, we append the authorization header to every request.
-        if ($withAuthorization) {
-            $authorizationHeader = $this->getAuthorizationHeader();
-            if (empty($options['headers'])) {
-                $options['headers'] = [];
-            }
+        // Find what traits this class is using.
+        $class = get_called_class();
+        $traits = array_keys(class_uses($class));
```

Comment:
Not a blocker for merge, but I noticed this will only get traits applied to the _exact_ class, not parent classes. So this array ends up empty for `LaravelNorthstarClient` for example, since the traits are applied on `NorthstarClient`. Can fix in another PR since it doesn't impact Rogue work.